test/k8s-e2e/e2e-test-execute

```diff
@@ -67,7 +67,8 @@ if [[ "$TESTDESCRIPTION_NAME" == "conformance.desc" ]]; then
 	conformance_test=$TRUE
 	echo -e "$(info INFO:) runnning Conformance e2e tests"
 fi
-if [[ "$TESTDESCRIPTION_NAME" =~ ^[[:digit:]]\.[[:digit:]]{1,2}$ ]]; then
+set +u
+if [ "$PUBLISH_TO_TESTGRID" = "true" ]; then
```

Comment:
Maybe use same `true` as for other tests?

Reply:
Done